src/packages/core/extension-registry/models/translations.model.ts

```diff
@@ -0,0 +1,26 @@
+import type { ManifestDefaultExport } from '@umbraco-cms/backoffice/extension-api';
+
+export interface ManifestTranslations extends ManifestDefaultExport<Record<string, Record<string, string>>> {
+	type: 'translations';
+	meta: MetaTranslations;
+}
+
+export interface MetaTranslations {
+	/**
+	 * The culture of the translations.
+	 * @example "en-US"
+	 */
+	culture: string;
+
+	/**
+	 * The translations.
+	 * @example
+	 * {
+	 *   "general": {
+	 *     "cancel": "Cancel",
+	 *     "close": "Close"
+	 *   }
+	 * }
+	 */
+	translations?: Record<string, Record<string, string>>;
+}
```

Comment:
This doesn't allow adding translations for multiple cultures in a single manifest.

Reply:
This is on purpose. You have to define the culture in your manifest. If you want to define multiple cultures, you simply register another manifest, which you can do even in the same file.